_includes/people/holdgraf.c.html

```diff
@@ -1,12 +1,12 @@
 <div class="media">
   <img class="media-object pull-left" src="{{page.root}}/img/people/holdgraf.c.jpg" alt="Chris Holdgraf" />
   <div id="holdgraf.c" class="media-body">
-    <span class="person">Chris Holdgraf</span>
+    <a href="chrisholdgraf.com"><span class="person">Chris Holdgraf</span></a>
```

Comment:
how's that?

On Thu, Feb 5, 2015 at 1:40 PM, Greg Wilson <notifications@github.com>
wrote:

> In _includes/people/holdgraf.c.html
> <https://github.com/swcarpentry/site/pull/771#discussion_r24202284>:
>
> > @@ -1,12 +1,12 @@
> >  <div class="media">
> >    <img class="media-object pull-left" src="{{page.root}}/img/people/holdgraf.c.jpg" alt="Chris Holdgraf" />
> >    <div id="holdgraf.c" class="media-body">
> > -    <span class="person">Chris Holdgraf</span>
> > +    <a href="chrisholdgraf.com"><span class="person">Chris Holdgraf</span></a>
>
> Please use <a class="person" href="...">your name</a>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/site/pull/771/files#r24202284>.
>